### PR TITLE
Singular iOS v2 device mode updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Singular iOS v2 device mode updates

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Singular iOS v2 device mode updates](https://www.notion.so/rudderstacks/Singular-device-mode-updates-4340707a42884edba22ee93dc58337da)

### CODE REVIEW

File: docs/destinations/analytics/singular.mdx [hunk 1]
```diff
@@ -97,6 +98,76 @@ RSClient *rudderClient = [RSClient getInstance:<WRITE_KEY> config:[configBuilder
...
+</ol>
+      </TabPanel>
+      <TabPanel>
+<div class="warningBlock">
+
+This device mode integration is supported for Singular v11.0.4 and above.
+</div>
+        To add Singular to your iOS app, follow these steps:
+<ol>
+<li>Install <code class="inline-code">RudderSingular</code> (available through <a href="https://cocoapods.org/">CocoaPods</a> by adding the following line to your <code class="inline-code">Podfile</code>:
```
Comment (ishachopra0401) [inline docs/destinations/analytics/singular.mdx:110]:
```suggestion
<li>Install <code class="inline-code">RudderSingular</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>:
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/analytics/singular.mdx [hunk 2]
```diff
@@ -97,6 +98,76 @@ RSClient *rudderClient = [RSClient getInstance:<WRITE_KEY> config:[configBuilder
...
+</span>
+
+<span>
+
+```objectivec
+@import RudderSingular;
+```
+</span>
+</li>
+<li>Next, add the imports to your <code class="inline-code">AppDelegate</code> file under <code class="inline-code">didFinishLaunchingWithOptions</code> method, as shown:
```
Comment (pallabmaiti) [inline docs/destinations/analytics/singular.mdx:134]: Shouldn't it be 'add the following code to your.......'?

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

File: docs/destinations/analytics/singular.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/destinations/analytics/singular.mdx:134]: We're keeping it consistent with the steps present in the Firebase doc.

Comment (ameypv-rudder) [inline docs/destinations/analytics/singular.mdx:134]:
```suggestion
<li>Next, add the imports to your <code class="inline-code">AppDelegate</code> file under the <code class="inline-code">didFinishLaunchingWithOptions</code> method, as shown:
```

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**